### PR TITLE
Validate pnts header

### PR DESCRIPTION
**Pull request by JudyWeng**
#24 

Please let me know if anything needs to be changed.

### CODE REVIEW

File: validator/lib/validatePnts.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            result : false,
+            message: 'Tile has the wrong byteLength'
+        };
+    }
+
+    return {
+        result : true,
+        message: 'Tile is a valid pnts tile'
+    };
+}
```
Comment (lilleyse) [inline validator/lib/validatePnts.js]: Add newline

File: validator/specs/lib/validatePntsSpec.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
+'use strict';
+var validatePnts = require('../../lib/validatePnts');
+
+describe('validatePnts', function() {
+
```
Comment (lilleyse) [inline validator/specs/lib/validatePntsSpec.js]: Remove empty line

File: validator/specs/lib/validatePntsSpec.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    it('returns true if the pnts tile is valid, returns false if the pnts has invalid magic', function() {
```
Comment (lilleyse) [inline validator/specs/lib/validatePntsSpec.js]: Same comment as the other PR about the wording.

File: validator/specs/lib/validatePntsSpec.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    header.writeUInt32LE(0, 12); // featureTableJSONByteLength
+    header.writeUInt32LE(0, 16); // featureTableBinaryByteLength
+    header.writeUInt32LE(0, 20); // batchTableJSONByteLength
+    header.writeUInt32LE(0, 24); // batchTableBinaryByteLength
+
+    return header;
+}
+
+function createInvalidVersion() {
+    var header = new Buffer(24);
```
Comment (lilleyse) [inline validator/specs/lib/validatePntsSpec.js]: 24 should be 28 (same below). Make sure to run the tests to catch these errors before hand.

File: validator/specs/lib/validatePntsSpec.js [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    header.write('pnts', 0); // magic
+    header.writeUInt32LE(1, 4); // version
+    header.writeUInt32LE(header.length - 1, 8); // byteLength
+    header.writeUInt32LE(0, 12); // featureTableJSONByteLength
+    header.writeUInt32LE(0, 16); // featureTableBinaryByteLength
+    header.writeUInt32LE(0, 20); // batchTableJSONByteLength
+    header.writeUInt32LE(0, 24); // batchTableBinaryByteLength
+
+    return header;
+}
```
Comment (lilleyse) [inline validator/specs/lib/validatePntsSpec.js]: Add newline.

**Comment by JudyWeng** [comment]
I did address these comments, but have yet to push because I followed similar pull/rebasing steps as #27 and I want to make sure I don't have additional old commits copied over like before

**Comment by lilleyse** [comment]
Ok. When all else fails its fine to create a new branch off of validator-master, cherry-pick the commits from this PR onto the new branch, set the upstream of the new branch so it faces this PR: `git branch --set-upstream-to origin/validate-pnts validate-pnts-new`, then force push.

**Comment by JudyWeng** [comment]
I went back and merged in place of all the rebased changes. Please let me know if this is okay and if there is anything that needs to be changed

File: validator/specs/lib/validatePntsSpec.js [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+function createWrongByteLength() {
+    var header = new Buffer(24);
```
Comment (lilleyse) [inline validator/specs/lib/validatePntsSpec.js]: I think you missed this one. Should be 28.

**Comment by lilleyse** [comment]
All else looks good.

**Comment by JudyWeng** [comment]
I just updated the spec file, sorry about that.

**Comment by lilleyse** [comment]
No problem! Looks great!

**Pull request merged**